pkg/sbctl/compatibility.go

```diff
@@ -205,9 +205,6 @@ func wrapListData(resource string, data []byte) ([]byte, error) {
 	case "storageclasses":
 		kind = "StorageClassList"
 		apiVersion = "storage.k8s.io/v1"
-	case "customresourcedefinitions":
```

Comment:
This is unexpected.

Reply:
Resolved this. This unexpected situation looks to be created during the PR. In the end no changes to `compatibility.go` were needed.